src/toil_scripts/batch_alignment/bwa_alignment.py

```diff
@@ -64,18 +72,23 @@ def parse_config(job, shared_ids, inputs):
     :param dict shared_ids: stores FileStoreIDs
     :param Namespace inputs: Input arguments
     """
-    samples = []
-
-    with open(inputs.config, 'r') as f_in:
-        for line in f_in:
-            if line.strip():
-                line = line.strip().split(',')
-                require(len(line) == 3, 'Improper formatting. Expected UUID,URl1,URL2. Received: {}'.format(line))
-                uuid = line[0]
-                urls = line[1:]
-                mock_bam = '.'.join(line[1].split('.')[:-2])[:-2] + ".bam"
-                samples.append((uuid, urls, mock_bam))
-    inputs.maxCores = int(inputs.maxCores) if inputs.maxCores else sys.maxint
+    if inputs.samples:
```

Comment:
I just used CSV because that's what was already being used. Should I go ahead with switching to YAML for the manifest? John's currently using TSV iirc.